packages/core/test/bundling/core_all/bundle.golden_size_map.json

```
@@ -0,0 +1,816 @@
+{
+  "unmapped": 25,
+  "files": {
+    "size": 267273,
+    "@angular/": {
+      "size": 247422,
+      "core/": {
+        "size": 247422,
+        "src/": {
+          "size": 247341,
+          "application_init.ts": {
```

if it is a file can we just have value rather than object literal (It is just noise).

```suggestion
          "application_init.ts": 626
```